### PR TITLE
New recipe for InSilicoSeq

### PR DESCRIPTION
**Pull request by standage**
Recipe for new package InSilicoSeq.

Note: I'm not the author (that honor belongs to @HadrienG), but I'm a happy user. 😀

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
Hmm, it's not building anything for some reason.

**Comment by standage** [comment]
> *That "a" will get you every time. *

Groan...🤨

**Comment by dpryan79** [comment]
I can't tell you how many times the yaml/yml thing has gotten me... On the plus side, at least the biocondabot indicated that nothing would be uploaded.

File: recipes/insilicoseq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - iss
+  requires:
+    - nose
```
Comment (dpryan79) [inline recipes/insilicoseq/meta.yaml]: You can probably get rid of `nose`, if it were actually required to import `iss` it should be a dependency.

Comment (standage) [reply recipes/insilicoseq/meta.yaml]: Done.

**Comment by dpryan79** [comment]
All green and it's getting built, so I'll merge. Nicely done @standage !

**Comment by dpryan79** [comment]
Give it 30 minutes before you hit rerun on the PR that depends on this.

**Comment by HadrienG** [comment]
Author here! Thanks for doing this! 🎉

**Comment by standage** [comment]
@HadrienG I would love to say I did this purely out of the goodness of my heart, but in reality I had selfish motives. I have a bioconda package that depends on InSilicoSeq. 😀

**Pull request merged**